### PR TITLE
PDPDEVTOOL-5018 - Unit Testing Stubs 4 

### PR DESCRIPTION
**Pull request by vic-nik**

### CODE REVIEW

File: packages/unit-testing/stubs/error/SuiteScriptError.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+define([], function () {
```
Comment (lfpvillegas) [inline packages/unit-testing/stubs/error/SuiteScriptError.js:1]: Should these new modules be added to SuiteCloudJestConfiguration.js ?

File: packages/unit-testing/stubs/format/i18n/NumberFormatter.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+         * @type {string}
+         * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
+         */
+        this.locale = undefined;
+        /**
+         * @name NumberFormatter#presicion
+         * @type {number}
+         * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
+         */
+        this.presicion = undefined;
```
Comment (lfpvillegas) [inline packages/unit-testing/stubs/format/i18n/NumberFormatter.js:26]: typo precision

File: packages/unit-testing/stubs/translation/Handle.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+define([], function() {
```
Comment (lfpvillegas) [inline packages/unit-testing/stubs/translation/Handle.js:1]: Missing Translator.js object?

**Review by antoniofruci — APPROVED** [review]

File: packages/unit-testing/stubs/format/i18n/NumberFormatter.js (hunk 2 above)
Comment (davidecorreu) [reply packages/unit-testing/stubs/format/i18n/NumberFormatter.js:26]: Fixing.

File: packages/unit-testing/stubs/translation/Handle.js (hunk 3 above)
Comment (davidecorreu) [reply packages/unit-testing/stubs/translation/Handle.js:1]: It goes the other way around, Handle.js is injected into translation.js.

**Review by lfpvillegas — APPROVED** [review]

**Review by lfpvillegas — APPROVED** [review]

**Pull request merged**